.circleci/config.yml

```diff
@@ -4,6 +4,8 @@ jobs:
     working_directory: ~/qpp-conversion-tool
     docker:
       - image: circleci/openjdk:8-jdk-browsers
+    environment:
+      MAVEN_OPTS: "-Xms1G -Xmx1G"
```

Comment:
We've been allocated the large resource pool now which gives us 4 cores and 8192 MB RAM.  Perhaps set `Xmx` to `7G`?

Reply:
We can try larger amounts later - this passes just fine in pretty much the same time, and circle isn't killing it. Better to make sure all the other builds are passing first before we mess around with this further imo.

Reply:
Agreed.